src/Assignment-5/assignment5_test.js

```diff
@@ -1,125 +1,78 @@
 // Copyright (c) 2015 Titanium I.T. LLC. All rights reserved. For license, see "README" or "LICENSE" file.
 
 // Example Quixote unit tests. We're using Mocha as our test framework and Chai for assertions.
-// These tests check the our media object CSS, which is defined in `screen.css`.
+// These tests check the our media object CSS, which is defined in `assignment2.css`.
```

Comment:
@satya-9 Please validate these test cases. I will wait for your confirmation before approving this PR.

Reply:
In previous test cases, the content was checked for top, left alignment w.r.t body, as the students approaches different mechanisms to align content at the center, we made test cases more generic.

Reply:
@sailesh-ramayanam I tested the testcases using the assignmnet-5 html and css of [https://github.com/wellimaj/assignment-5/tree/assignment-5] github repo and it ran successfully.

Reply:
@satya-9 Is wellimaj among the students who complained that test cases were not passing?

Reply:
@sailesh-ramayanam 

1. Yes sailesh.wellimaj is among those students who complained
2. Actually there were no students who passed all the testcases.
3. Yes i checked the solution which was given by mayank and it passed